FifthLesson/VectorProgram/VectorProgram/ProgramRun.cs

```diff
@@ -13,87 +9,132 @@ class ProgramRun
     {
         public void Run()
         {
-            Vector vectorFirst = EnterUserData("first");
-            Vector vectorSecond = EnterUserData("second");
+            Vector vectorFirst = GetUserVector("first");
+            Vector vectorSecond = GetUserVector("second");
+
+            ShowVectors(vectorFirst, vectorSecond);
+
+            ShowActionsWithVectors(vectorFirst, vectorSecond);
 
-            ShowVectorsAndActions(vectorFirst, vectorSecond);
             ShowMultiplicationAndVectors(vectorFirst, vectorSecond);
+
             CompareVectors(vectorFirst, vectorSecond);
 
-            Console.WriteLine($"The angle between {vectorFirst.ToString()} and {vectorSecond.ToString()} = {(vectorFirst.AngleBetweenVectors(vectorSecond)).ToString()}");
+            ShowVectorMultiplication(vectorFirst, vectorSecond);
+
+            ShowAngleBetweenVectors(vectorFirst, vectorSecond);
+
+            ReadKey();
+        }
 
-            Console.ReadKey();
+        private void ShowAngleBetweenVectors(Vector vectorFirst, Vector vectorSecond)
+        {
+            var angle = vectorFirst.AngleBetweenVectors(vectorSecond);
+            WriteLine($"The angle between {vectorFirst.ToString()} and {vectorSecond.ToString()} = {angle.ToString()}");
         }
 
-        private static void CompareVectors(Vector vectorFirst, Vector vectorSecond)
+        private void CompareVectors(Vector first, Vector second)
         {
-            Console.WriteLine($"{vectorFirst.ToString()} == {vectorSecond.ToString()} = {(vectorFirst == vectorSecond).ToString()}");
-            Console.WriteLine($"{vectorFirst.ToString()} != {vectorSecond.ToString()} = {(vectorFirst != vectorSecond).ToString()}");
+            WriteLine("Compare vectors:");
+            WriteLine($"{first.ToString()} == {second.ToString()} = {(first == second).ToString()}");
```

Comment:
**ToString** will be called by default.
Please, don't type it.

Reply:
Corrected.